### PR TITLE
feat: reading statistics for member page

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 個人閱讀紀錄-會員頁UI更新](https://app.asana.com/0/1203699264568808/1205830825368922/f)

# Notice
- can use `?hideInfo=true&articleReadCount=1000&readingSec=123` params for display

# Dependency
[npm-package#451](https://github.com/twreporter/twreporter-npm-packages/pull/451)

### CODE REVIEW

File: src/components/member-page/member-data.js [hunk 1]
```diff
@@ -33,12 +51,36 @@ const BottomDividerContainer = styled.div`
...
+            <H1 text={articleReadCount.toLocaleString()} />
+            <P1 text={'篇'} />
+          </ReadingStatistics>
+          <ReadingStatistics>
+            <P1 text={'閱讀時間'} />
+            <H1
+              text={
+                articleReadingTime > 99999
+                  ? '99,999+'
+                  : articleReadingTime.toLocaleString()
```
Comment (Aylie-Chou) [inline src/components/member-page/member-data.js:77]: Could we specify the `locale` parameter for `toLocaleString` func ?

File: src/containers/Member.js [hunk 2]
```diff
@@ -148,6 +149,38 @@ const MemberPage = ({
...
+      return {
+        articleReadingTime: days,
+        articleReadingTimeUnit: READING_TIME_UNIT.day,
+      }
+    }
+  }
+
+  // TODO: remove after get data from api
+  // ?hideInfo=true&articleReadCount=1000&readingSec=123
+  const param = new URLSearchParams(search)
```
Comment (Aylie-Chou) [inline src/containers/Member.js:176]: Just a bonus:
we could also use [`useParams`](https://v5.reactrouter.com/web/api/Hooks/useparams) to get url params by:
```
const { hideInfo } = useParams()

File: src/components/member-page/member-data.js (hunk 1 above)
Comment (liruchen32) [reply src/components/member-page/member-data.js:77]: fixed

**Review by Aylie-Chou — APPROVED** [review]
ＬＧＴＭ

**Pull request merged**